### ISSUE TITLE
Salutations in Core vereinheitlichen

### ISSUE THREAD

**Issue by carlobeltrame** [hitobito/hitobito#1141]
Bisher haben folgende Wagons "Anrede"-Felder für Personen:
- PBS
- CVP (kopiert von der PBS)
- DSJ
- CEVI
- insieme

Die Implementationen könnten vereinheitlicht werden und in den Core verschoben werden.

**Comment by Michael-Schaer** [comment]
Ja, das macht Sinn.

Bei der Gelegenheit würde ich das dritte Geschlecht noch mal einwerfen. Für uns wird das womöglich im 2021 noch ein Thema sein. Wäre es besser, das zusammen zu lösen, da es doch recht mit der Anrede zusammenhängt?

**Comment by carlobeltrame** [comment]
Ja, zumindest sollte die vereinte Anrede-Lösung flexibel genug dafür sein. Das dritte Geschlecht war bei der Jubla auch schon ein Thema, hat aber unter Umständen noch an mehr Orten Auswirkungen als nur bei der Anrede.

**Comment by ThomasEllenberger** [comment]
> Ist das jetzt nicht im Core?
> 
> Siehe z.B. hier: [hitobito/hitobito_pbs#198](https://github.com/hitobito/hitobito_pbs/pull/198)

Habe dies kurz in einer Tenants Umgebung getestet. Im Core erscheinen noch keine Anreden.
